test/libsolidity/util/SoltestTypes.h

```diff
@@ -229,6 +230,32 @@ struct FunctionCallArgs
 	}
 };
 
+/// Units that can be used to express function value
+enum class FunctionValueUnit
+{
+	Wei,
+	Ether
+};
+
+/// FunctionValue structure
```

Comment:
```suggestion
/// FunctionValue structure
```
Please remove this line, it is redundant.